deployment/eks/flyte_aws_scheduler_helm_generated.yaml

```diff
@@ -1208,7 +1208,7 @@ spec:
         imagePullPolicy: "IfNotPresent"
         name: datacatalog
         ports:
-        - containerPort: 8088
+        - containerPort: 8080
```

Comment:
could we change the default port back to 8088?

Reply:
yes, I know. I mean, we should change this [port](https://github.com/flyteorg/flyte/blob/ba5925868f9ece84c834a9d5a1e09f6287394bce/charts/flyte-core/values.yaml#L692) to 8088

Reply:
ah, I see, so 8088 never works. we use 8080 in the [configmap](https://github.com/flyteorg/flyte/blob/caed58456f3039840081bf42f7775500b542a36b/charts/flyte-core/values.yaml#L692), but use 8088 for container port. 
@ddl-ebrown Thanks for investigating and fixing it.